core/store/src/trie/trie_recording.rs

```diff
@@ -360,7 +360,7 @@ mod trie_recording_tests {
             // Now let's do this again with memtries enabled. Check that counters
             // are the same.
             assert_eq!(MEM_TRIE_NUM_LOOKUPS.get(), mem_trie_lookup_counts_before);
-            tries.load_mem_trie(&shard_uid, None).unwrap();
+            tries.load_mem_trie(&shard_uid, None, false).unwrap();
```

Comment:
Do we actually have cases where non-parallel loading could be beneficial? Except for correctness tests.

Reply:
Yes. We should not parallel load during state sync when it is not desired to saturate all the cores.